config/config.conf

```diff
@@ -1,18 +1,56 @@
-monitoringCluster="cagg"
+# Pipelines
+# ~~~~
+hostPipelineConfiguration="config/hostPipeline.conf"
+clusterPipelineConfiguration="config/clusterPipeline.conf"
+
+# Http Server
+# ~~~~
+#httpHost="0.0.0.0"
+#httpPort=7066
+#httpHealthCheckPath="/ping"
+#httpStatusPath="/status"
+#httpVersionPath="/version"
+
+# Metrics
+# ~~~~
+monitoringCluster="cluster_aggregator"
+#monitoringService="cluster_aggregator"
+#monitoringSinks=[
+#  {
+#    class = "com.arpnetworking.metrics.impl.ApacheHttpSink"
+#    bufferSize = 10000
+#    uri = "http://localhost:7090/metrics/v3/application"
+#    parallelism = 2
+#    maxBatchSize = 500
+#    emptyQueueInterval = "PT0.5S"
+#    eventsDroppedLoggingInterval = "PT1M"
+#    dispatchErrorLoggingInterval = "PT1M"
+#    unsupportedDataLoggingInterval = "PT1M"
+#  }
+#]
+#jvmMetricsCollectionInterval="PT1.0S"
+
+# NOTES:
+# - monitoringHost and monitoringPort are deprecated
+# - specifying monitoringHost or monitoringPort disables monitoringSinks
```

Comment:
this is missing a single line from the version in MAD:
```- metricsClientHost and metricsClientPort no longer default from httpHost and httpPort```

Reply:
So that behavior was specific to MAD because the client wants to send its data to MAD -- yes, MAD talks to itself by default; how crazy is that? ;)